torch/tensor.py

```diff
@@ -425,8 +425,12 @@ def __contains__(self, element):
         """
         if isinstance(element, (torch.Tensor, Number)):
             return (element == self).any().item()
-        return NotImplemented
 
+        raise RuntimeError(
+            "Tensor.__contains__ only supports Tensor or scalar, but you passed in a %s." %
+            type(element)
+        )
+        
```

Comment:
our linter is complaining: ./torch/tensor.py:433:1: W293 blank line contains whitespace

https://dev.azure.com/pytorch/PyTorch/_build/results?buildId=4501

You can get to the results of the linter by clicking through the "Python 2.7 Lint" or "Python 3.5 Lint below"

Reply:
The message would also be better if it said "Number" instead of "scalar"

Reply:
okay~